src/CMakeLists.txt

```diff
@@ -47,5 +50,7 @@ set(XENBE_LIB xenbe)
 target_link_libraries(${PROJECT_NAME}
 	${XENBE_LIB}
 	${V4L2_LIBRARY}
+	${MEDIACTL_LIBRARIES}
+	v4l2subdev
```

Comment:
Why this doesn't go into MEDIACTL_LIBRARIES? My understanding is that we didn't require this before and this is MEDIACTL_LIBRARIES which requires it

Reply:
because this is not a part of MEDIACTL_LIBRARIES. Although provided by media-ctl, this is a separate library

Reply:
ok